<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1066</width>
    <height>782</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fichaje de trabajadores</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">
      background-color: #2E2E2E;
      color: #FFFFFF;
      font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
    </string>
   </property>
   <widget class="QFrame" name="menu_frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>200</width>
      <height>782</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
       background-color: #1C1C1C;
       border-right: 1px solid #444;
     </string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="fichar_boton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">
          QPushButton {
            background-color: #4CAF50;
            color: white;
            border: none;
            padding: 10px 20px;
            text-align: center;
            text-decoration: none;
            display: inline-block;
            font-size: 16px;
            margin: 4px 2px;
            cursor: pointer;
            border-radius: 12px;
          }
          QPushButton:hover {
            background-color: #45a049;
          }
        </string>
       </property>
       <property name="text">
        <string>Fichar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="imprimir_boton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">
          QPushButton {
            background-color: #008CBA;
            color: white;
            border: none;
            padding: 10px 20px;
            text-align: center;
            text-decoration: none;
            display: inline-block;
            font-size: 16px;
            margin: 4px 2px;
            cursor: pointer;
            border-radius: 12px;
          }
          QPushButton:hover {
            background-color: #007bb5;
          }
        </string>
       </property>
       <property name="text">
        <string>Imprimir</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QStackedWidget" name="content_frame">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>0</y>
      <width>861</width>
      <height>541</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
       background-color: #3E3E3E;
       border-radius: 10px;
       padding: 10px;
     </string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLineEdit" name="codigo_placeholder">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>320</y>
        <width>137</width>
        <height>40</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">
         background-color: #8080FF;
         color: white;
         border: 1px solid #444;
         border-radius: 5px;
         padding: 5px;
       </string>
      </property>
     </widget>
     <widget class="QPushButton" name="emitir_fichaje">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>429</y>
        <width>141</width>
        <height>61</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>60</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">
         QPushButton {
           background-color: #FF5722;
           color: white;
           border: none;
           padding: 10px 20px;
           text-align: center;
           text-decoration: none;
           display: inline-block;
           font-size: 16px;
           margin: 4px 2px;
           cursor: pointer;
           border-radius: 12px;
         }
         QPushButton:hover {
           background-color: #E64A19;
         }
       </string>
      </property>
      <property name="text">
       <string>Emitir Fichaje</string>
      </property>
     </widget>
     <widget class="QLabel" name="teclea_codigo_label">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>270</y>
        <width>209</width>
        <height>40</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">
         font: 16pt &quot;Segoe UI&quot;;
         color: white;
       </string>
      </property>
      <property name="text">
       <string>Teclea tu Codigo</string>
      </property>
     </widget>
     <widget class="QLabel" name="current_time">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>70</y>
        <width>209</width>
        <height>40</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">
         font: 16pt &quot;Segoe UI&quot;;
         color: white;
       </string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="text">
       <string>Fecha</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="current_hour">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>70</y>
        <width>209</width>
        <height>40</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">
         font: 16pt &quot;Segoe UI&quot;;
         color: white;
       </string>
      </property>
      <property name="text">
       <string>Hora</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QListWidget" name="list_checkins">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>300</y>
        <width>311</width>
        <height>192</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
         background-color: #FFFFFF;
         border: 1px solid #444;
         color: #000;
       </string>
      </property>
     </widget>
     <widget class="QDateTimeEdit" name="date_start">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>110</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">
         font: 14pt &quot;Segoe UI&quot;;
         color: #FFB41D;
       </string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QDateTimeEdit" name="date_end">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>110</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">
         font: 14pt &quot;Segoe UI&quot;;
         color: #FFB41D;
       </string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>14</x>
        <y>250</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI,Tahoma,Geneva,Verdana,sans-serif</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">
         color: #FFB41D;
       </string>
      </property>
      <property name="text">
       <string>Estado Trabajadores</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>60</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI,Tahoma,Geneva,Verdana,sans-serif</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">
         color: #FFB41D;
       </string>
      </property>
      <property name="text">
       <string>Fecha Desde</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>60</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI,Tahoma,Geneva,Verdana,sans-serif</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">
         color: #FFB41D;
       </string>
      </property>
      <property name="text">
       <string>Fecha Hasta</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="imprimir_button">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>340</y>
        <width>161</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI,Tahoma,Geneva,Verdana,sans-serif</family>
        <pointsize>-1</pointsize>
        <underline>false</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">
         QPushButton {
           background-color: #FF5722;
           color: white;
           border: none;
           padding: 10px 20px;
           text-align: center;
           text-decoration: none;
           display: inline-block;
           font-size: 16px;
           margin: 4px 2px;
           cursor: pointer;
           border-radius: 12px;
         }
         QPushButton:hover {
           background-color: #E64A19;
         }
       </string>
      </property>
      <property name="text">
       <string>Imprimir</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="lower_frame">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>550</y>
      <width>861</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">       background-color: #3E3E3E;
       border-radius: 10px;
       padding: 10px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="1">
      <widget class="QPushButton" name="accept_button">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">
          QPushButton {
            background-color: #4CAF50;
            color: white;
            border: none;
            padding: 10px 20px;
            text-align: center;
            text-decoration: none;
            display: inline-block;
            font-size: 16px;
            margin: 4px 2px;
            cursor: pointer;
            border-radius: 12px;
          }
          QPushButton:hover {
            background-color: #45a049;
          }
        </string>
       </property>
       <property name="text">
        <string>Accept</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="reject_button">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">
          QPushButton {
            background-color: #F44336;
            color: white;
            border: none;
            padding: 10px 20px;
            text-align: center;
            text-decoration: none;
            display: inline-block;
            font-size: 16px;
            margin: 4px 2px;
            cursor: pointer;
            border-radius: 12px;
          }
          QPushButton:hover {
            background-color: #D32F2F;
          }
        </string>
       </property>
       <property name="text">
        <string>Reject</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" rowspan="2">
      <widget class="QTextEdit" name="show_info">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>170</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">
          background-color: #F0FFFF;
          color: #000;
          border: 1px solid #444;
        </string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
